app/views/admin/conference_domains/show.haml

```diff
@@ -0,0 +1,95 @@
+.row
```

Comment:
The instructions in this file are the same as in conference_domains/_form.haml right? Maybe you can extract this into a partial?

Reply:
There is a difference between the two. The `conference_domains/_form` partial contains a form in step 2 where the domain name can be added whereas the help text is just simple text with no such form. 

* Help text

![screenshot_20170921_220653](https://user-images.githubusercontent.com/14155445/30707513-3b4b86e8-9eeb-11e7-99d0-ecc85710b30d.png)

* _form.haml 

![screenshot_20170921_220715](https://user-images.githubusercontent.com/14155445/30707512-3b3d258a-9eeb-11e7-9eac-419836010646.png)

This is the reason why I preferred not to extract it into a partial.

Reply:
How about the extracting each step into its own partial and having 2 different "step 2" partials?